pkg/controllers/dynakube/status/status.go

```diff
@@ -8,7 +8,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-func SetDynakubeStatus(ctx context.Context, dynakube *dynatracev1beta1.DynaKube, apiReader client.Reader) error {
+func SetKubeSystemUUIDInStatus(ctx context.Context, dynakube *dynatracev1beta1.DynaKube, apiReader client.Reader) error {
```

Comment:
imho we should have the same identifier everywhere- I would prefer SetDynakubeSystemUUIDInStatus here

Reply:
I think its redundant to say `Dynakube`, also its the `KubeSystemUUID` we want to add to the `Dynakube`'s status, so if we want to be super explicit then we would need to say something like `SetKubeSystemUUIDInDynakubeStatus` which is a mouthful 😅

Reply:
true from that perspective this makes sense.